js/libdeno.ts

```diff
@@ -18,6 +24,9 @@ interface Libdeno {
 
   shared: ArrayBuffer;
 
+  /* tslint:disable-next-line:no-any */
+  eval(code: string): [any, EvalErrorInfo | null];
```

Comment:
I think this is a good place to add some JSDOC explaining its behavior in detail.